src/components/Common/Navbar2.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const logout = () => {
```

Comment:
Why are there two of these?

Reply:
i did not want to scope creep but this navbar features need to be converted into SCSS instead of the bootstrap that is there now. - i assumed it would be another ticket so i pushed both to edit later.